src/cardproduct.css

```diff
@@ -0,0 +1,97 @@
+.card-container {
+  display: flex;
+  font-family: Roboto, sans-serif;
+  font-style: normal;
+}
+
+.card-product {
```

Comment:
please add paddings for `.card-product` instead setting margins to all elements inside card